### PR TITLE
bump version to 1.6.1.0 for Dnn v9.2.2 rc0 build

### PR DESCRIPTION
**Pull request by ohine**
version bump to v1.6.1.0 for generating dnn v9.2.2 rc package

### CODE REVIEW

File: Build/Symbols/Symbols.dnn [hunk 1]
```diff
@@ -1,6 +1,6 @@
-﻿<dotnetnuke type="Package" version="5.0">
+<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="Dnn.PersonaBar.UI_Symbols" type="Library" version="07.01.02">
+    <package name="Dnn.PersonaBar.UI_Symbols" type="Library" version="01.06.01">
```
Comment (mitchelsellers) [inline Build/Symbols/Symbols.dnn:3]: This change will result in an install issue as it is making the version # lower than that of the installed one

File: Extensions/Build/Symbols/Symbols.dnn [hunk 2]
```diff
@@ -1,6 +1,6 @@
-﻿<dotnetnuke type="Package" version="5.0">
+<dotnetnuke type="Package" version="5.0">
   <packages>
-    <package name="Dnn.PersonaBar.Extensions_Symbols" type="Library" version="07.01.02">
+    <package name="Dnn.PersonaBar.Extensions_Symbols" type="Library" version="01.06.01">
```
Comment (mitchelsellers) [inline Extensions/Build/Symbols/Symbols.dnn:3]: Similar comment on version number here

**Review by mitchelsellers — CHANGES_REQUESTED** [review]
Looks like two of the packages need to follow a different version # format.  Possibly just make them 9.2.2?

File: Extensions/Build/Symbols/Symbols.dnn (hunk 2 above)
Comment (ohine) [reply Extensions/Build/Symbols/Symbols.dnn:3]: Was the symbol package ever published? We don't have any published releases of the personabar, and the symbols don't appear to be on any previous release of platform... right? Or am I missing something? They'll be one more commit pushed to update the nuspec file version nodes in a few minutes....

**Review by mitchelsellers — APPROVED** [review]
Based on the conversation, the symbols have not previously been published this change isn't impacting the greater community, approved.

Future releases should look at using a different Version number format.

**Comment by ohine** [comment]
Just pushed a commit with updated nuspec files. That should be everything required. I also updated the powershell script which I'm using via VSTS to set these version numbers for the next release.

**Review by mitchelsellers — APPROVED** [review]

**Pull request merged**